### PR TITLE
install & demo script

### PR DESCRIPTION
**Pull request by dacleyra**

### CODE REVIEW

**Comment by dacleyra** [comment]
@stephenkinder @kyleschlosser 

operator install script
tekton example pipeline demo script

File: ref/scripts/install-kabanero-foundation.sh [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+# Kiali - Optional#
+# bash <(curl -L https://git.io/getLatestKialiOperator) --operator-image-version v1.0.0 --operator-watch-namespace '**' --accessible-namespaces '**' --operator-install-kiali true || true
+
+# Istio #
+release=maistra-0.12.0
+oc new-project istio-operator || true
+oc new-project istio-system || true
+oc apply -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/${release}/deploy/maistra-operator.yaml
+oc apply -n istio-system -f https://raw.githubusercontent.com/Maistra/istio-operator/${release}/deploy/examples/maistra_v1_servicemeshcontrolplane_cr_minimal.yaml
```
Comment (stephenkinder) [inline ref/scripts/install-kabanero-foundation.sh]: Why are we using the Maistra operator? I believed we were using the operator at this project:  https://github.com/istio/operator.  The Maistra project doesn't have any LICENSE file, and I believe that the istio.io team is building at the latter project.

File: ref/scripts/install-kabanero-foundation.sh [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+do
+  sleep 1
+done
+
+
+
+### Tekton Dashboard ###
+
+# Webhook Extension #
+curl -L https://raw.githubusercontent.com/tektoncd/experimental/master/webhooks-extension/config/release/gcr-tekton-webhooks-extension.yaml \
```
Comment (stephenkinder) [inline ref/scripts/install-kabanero-foundation.sh]: Do we need a comment about why "the experimental" branch is being used?  Is it a bug patch?

File: ref/scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Sample Appsody project with manual Tekton pipeline run
+
+Create a Persistent Volume for the pipeline to use. A sample hostPath `pv.yaml` is provided.
+```
+oc apply -f pv.yaml
+```
+
+Create the pipeline and execute the example manual pipeline run
+```
+APP_REPO=https://github.com/dacleyra/appsody-hello-world/ ./appsody-tekton-example-manual-run.sh
```
[the remark this answers is not in the document]
Comment (dacleyra) [reply ref/scripts/README.md]: Neither org has a repo containing just the java-microprofile project result from an `appsody init`.

File: ref/scripts/install-kabanero-foundation.sh (hunk 1 above)
Comment (dacleyra) [reply ref/scripts/install-kabanero-foundation.sh]: This is now set as the knative istio install, until we have an official istio operator release

File: ref/scripts/install-kabanero-foundation.sh (hunk 2 above)
Comment (dacleyra) [reply ref/scripts/install-kabanero-foundation.sh]: these have PRs pending to be added to tekton operator
added comment indicating such

File: ref/scripts/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (stephenkinder) [reply ref/scripts/README.md]: Ok for now.  Given the timeframe, I'm inclined to let this go, since this is just an example.

**Pull request merged**